### ISSUE TITLE
incorrect search results

### ISSUE THREAD

**Issue by gubuntu** [kartoza/feti#68]
I searched for 'fibre processing' and result included at least one record without that string

 
![screenshot from 2015-06-03 14 58 40](https://cloud.githubusercontent.com/assets/496911/7960493/470afe4a-0a01-11e5-9ce0-7280c1509949.png)

And I searched for 'fibre' and got no results!

**Comment by lucernae** [comment]
This can happened for the following reasons (assuming we already indexed the models):

1. There was actually no string contains 'fibre', so we got zero results
2. The result you captured in the map is included because it contains the words process, which is the stem word of processing. Try it with "Operator", it also return results containing the word "Operations". Try it again with "Packaging"

For case number 2, I don't know if it can be seen as a feature or bug. It depends on what the user expects. If we want to disable stemming, then perhaps we should change the search analyzer in the elastic search configuration.
